src/analysis/specific_limits.lean

```diff
@@ -94,6 +97,18 @@ begin
   end
 end
 
+/-- The function `r ↦ r⁻¹` tends to `0` on the right as `r → +∞`. -/
+lemma tendsto_inv_at_top_zero' [discrete_linear_ordered_field α] [topological_space α]
```

Comment:
Not sure why you need a prime in the name here.

Reply:
OK, let me do this.